ross/bearing_seal_element.py

```diff
@@ -307,7 +310,7 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(self.tag)
 
-    def save(self, file_name):
+    def save(self, file_name=Path(os.getcwd())):
```

Comment:
If we have `file_name` with a default value, the user can create two different bearings and when saving we would overwrite the first bearing. Maybe we `file_name` should be a required positional argument?

Reply:
Sorry, I think I was not clear in my comment.
The previous code had `file_name` as a required positional argument, so if the user tried to call `.save()` without providing the `file_name` this would raise an exception, which is the expected behavior.
With the current code (`def save(self, file_name=Path(os.getcwd())):`, having `file_name` as a keyword argument with a default value, the user can call `.save()` without the `file_name` argument and it will save the file, overwriting anything that is already there, which is a behavior that the user does not expect (in other libraries `.save` methods you have to provide a path).
So, I was actually asking that `.save` methods had the signature `def save(file_name)`, instead of `def save(file_name=Path(os.getcwd()))`.

Reply:
Oh my mistake, I understand this now!

Reply:
Currently the code is looking like this:
```Python
 def save(self, file_name):
        """Saves a bearing element in a toml format. It works as an auxiliary
        function of the save function in the Rotor class.
        Parameters
        ----------
        file_name: string
            The name of the file the bearing element will be saved in.
        Returns
        -------
        None
        Examples
        --------
        >>> bearing = bearing_example()
        >>> bearing.save(Path(os.getcwd()))
        """
        data = self.get_data(Path(file_name)/'BearingElement.toml')
``` 
The code already complies with the specifications!